src/main/java/com/google/api/codegen/nodejs/NodeJSGapicContext.java

```diff
@@ -124,58 +122,77 @@ private String fieldComment(String comment, Field field) {
-    MethodConfig config = getApiConfig().getInterfaceConfig((Interface) method.getParent())
-        .getMethodConfig(method);
 
-    String callbackType =
-        isEmpty ? "EmptyCallback" : String.format("APICallback<%s>", classInfo);
-    String callbackComment = "@param {?" + callbackType + "} callback\n"
-        + "  The function which will be called with the result of the API call.";
+    String callbackType = isEmpty ? "EmptyCallback" : String.format("APICallback<%s>", classInfo);
+    String callbackComment =
+        "@param {?"
+            + callbackType
+            + "} callback\n"
+            + "  The function which will be called with the result of the API call.";
     if (config.isPageStreaming()) {
       String resourceType = jsTypeName(config.getPageStreaming().getResourcesField().getType());
-      return callbackComment + "\n@returns {?Stream<" + resourceType + ">}\n"
-          + "  An object stream of " + resourceType + " instances, unless\n"
+      return callbackComment
+          + "\n@returns {?Stream<"
+          + resourceType
+          + ">}\n"
+          + "  An object stream of "
+          + resourceType
+          + " instances, unless\n"
           + "  page streaming is disabled through the call options or a callback\n"
           + "  is specified. If page streaming is disabled or a callback is specified,\n"
           + "  this returns null, and the callback will be called with a single instance\n"
-          + "  of " + classInfo + ".";
+          + "  of "
+          + classInfo
+          + ".";
     } else {
       return callbackComment;
     }
   }
 
-
   /**
    * Return comments lines for a given method, consisting of proto doc and parameter type
    * documentation.
    */
   public List<String> methodComments(Method msg) {
-    MethodConfig config = getApiConfig().getInterfaceConfig((Interface) msg.getParent())
-        .getMethodConfig(msg);
+    MethodConfig config =
+        getApiConfig().getInterfaceConfig((Interface) msg.getParent()).getMethodConfig(msg);
 
     // Generate parameter types
     StringBuilder paramTypesBuilder = new StringBuilder();
     for (Field field : config.getRequiredFields()) {
-      paramTypesBuilder.append(fieldParamComment(field, false));
+      paramTypesBuilder.append(fieldParamComment(field, null, false));
     }
     Iterable<Field> optionalParams = config.getOptionalFields();
     if (optionalParams.iterator().hasNext()) {
       paramTypesBuilder.append("@param {?Object} otherArgs\n");
       for (Field field : optionalParams) {
-        paramTypesBuilder.append(fieldParamComment(field, true));
+        if (config.isPageStreaming()
+            && field.equals((config.getPageStreaming().getPageSizeField()))) {
+          paramTypesBuilder.append(
+              fieldParamComment(
+                  field,
+                  "The maximum number of resources contained in the underlying API\nresponse. If "
```

Comment:
nit: I like to break the line at "\n" instead of padding as far as possible -- so it can look same as what will appear in the actual code.